recipes/symengine/bld.bat

```diff
@@ -0,0 +1,19 @@
+mkdir build
+cd build
+
+cmake ^
+    -G "%CMAKE_GENERATOR%" ^
```

Comment:
Yes, you are right. This [change](https://github.com/conda/conda-build/commit/d2d5d8a62d719485717700637f0f8d2394bffa23) is not in `1.20.1`